### DISCUSSION TITLE
Very complicated and messy texts, descriptions. Totally incomprehensible material.

### DISCUSSION THREAD

**Discussion by alexejibishvili** [IntelliTect/EssentialCSharp.Web#392] [Feedback and Ideas]
I've lots of years of experience in .NET C#.
I came up to check the resource for sharing with juniors.
It's totally inconsequent material, starting from left side menu, its order, subjects, how they are separated.
Texts and examples in chapters are so messy, I cant logically understand what is next, what I'm learning right now.

IT CAN BE SIMPLIFIED RADICALLY!
PROVE ME WRONG!

**Comment by BenjaminMichaelis** [comment]
Hello @alexejibishvili,

Thank you for sharing your thoughts with us! We appreciate your feedback.

While this version of the website represents an initial effort to complement the physical book, rest assured that we have a roadmap of exciting updates ahead. Your input is invaluable, and we encourage you to provide specific suggestions if you have any on your mind. We’ll continue to enhance this resource, making it even more valuable for developers like yourself! 🚀

**Comment by BenjaminMichaelis** [comment]
If you have specific feedback, feel free to reach back out. Closing for now.
